### PR TITLE
feat: add project list command

### PR DESCRIPTION
**Pull request by Yangyang96**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., design docs, usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/project/storages/local_test.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
+package storages
+
+import (
+	"os"
+	"testing"
+)
+
+func TestNewLocalStorage(t *testing.T) {
+	// 定义测试用例
```
Comment (liu-hm19) [inline pkg/project/storages/local_test.go:9]: Please translate the comments into English.

File: pkg/project/storages/util_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		})
+	}
+}
+
+func TestGenGenericOssReleasePrefixKey(t *testing.T) {
+	tests := []struct {
+		prefix string
+		want   string
+	}{
+		{"", releasesPrefix},                             // 测试空字符串
```
Comment (liu-hm19) [inline pkg/project/storages/util_test.go:31]: Ditto.

File: pkg/project/storages/util_test.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package storages
+
+import (
+	"testing"
+)
+
+func TestGenProjectDirPath(t *testing.T) {
+	// 定义测试用例
```
Comment (liu-hm19) [inline pkg/project/storages/util_test.go:8]: Please translate the comments into English.

**Comment by Yangyang96** [comment]
Done fixing comments.

**Review by liu-hm19 — APPROVED** [review]
lgtm

**Pull request merged**